test/fw/ptl/utils/plugins/ptl_test_info.py

```diff
@@ -128,26 +128,9 @@ def wantClass(self, cls):
         """
         Is the class wanted?
         """
-        if not issubclass(cls, PBSTestSuite):
-            return False
-        if cls.__name__ == 'PBSTestSuite':
-            return False
         self._tree.setdefault(cls.__name__, cls)
         if len(cls.__bases__) > 0:
             self.wantClass(cls.__bases__[0])
-        return False
-
-    def wantFunction(self, function):
```

Comment:
was this dead code?

Reply:
Yes. This was not called anywhere.

Reply:
@agrawalravi90 It was not the dead code and @shilpakodli it was getting called by nose internal call. And test info plugin was designed such a way that it works without any other ptl node plugin but with this PR changes now we are using loader plugin along with info plugin and loader plugin already has wantFunction so we don't need it here.